CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.20.0] - 2020-05-20
+### Changed
+- Move to `reqwest` 0.10
+- Support of synchronous interface is removed
+- async/.await support has come
```

Comment:
```suggestion
## Unreleased
### Changed
- `async`/`.await` support has arrived! :tada:
- Upgraded `reqwest` to `0.10`.

### Removed
- Synchronous access has been removed in favor of `async`/`.await`.
```

When version is `<1.0.0`, minor version changes are treated as major ones. I don't think there's any need to bump to `0.20.0` just for the nice number. I will bump the version and add the date of release myself (so please undo changes to the version number in other places too), no need to rush :)